### DISCUSSION TITLE
Change tab height

### DISCUSSION THREAD

**Discussion by EvBear** [onemen/TabMixPlus#327] [Ideas]
Thanks for this great addon, I recently installed this, and was wondering if it would be possible to add a feature to re-size the height of the tabs to make them smaller?

**Comment by pyrates999** [comment]
look at https://github.com/MrOtherGuy/firefox-csshacks/tree/master/chrome/

to get started:

```
about:config -> toolkit.legacyUserProfileCustomizations.stylesheets -> true
create <user-profile-dir>/chrome/userChrome.css
```

and I think this allows you to set the height of tabs in `<user-profile-dir>/chrome/userChrome.css`:

```
:root {
  --tab-min-height: 28px !important;
  --tab-max-height: 28px !important;
  --tab-min-width: 120px !important;
  --tab-max-width: 120px !important;
  --arrowpanel-menuitem-padding: 4px 8px !important;
}
```

**Comment by EvBear** [comment]
Thanks, that workaround is just what I needed. Will I need to do this every time firefox updates?

**Reply by pyrates999** [reply]
No because this is overriding what firefox is using.  Although firefox may need to you change stuff in about:config in the future.  But for now, no.

**Comment by onemen** [comment]
in about:config set `browser.uidensity` to 1

**Reply by pyrates999** [reply]
thank you, it seems I already set this, but this wasn't documented in my userChrome.css file.
